contrib/IECoreUSD/test/IECoreUSD/USDSceneWriterTest.py

```diff
@@ -822,5 +822,17 @@ def testRefCountAfterWrite( self ) :
 			refCounts
 		)
 
+	def testUVs( self ) :
+
+		root = IECoreScene.SceneInterface.create( "/tmp/test.usda", IECore.IndexedIO.OpenMode.Write )
```

Comment:
I did see that mechanism, but I couldn't see the point in using it. It's not used consistently in that file anyway, it's not available in USDSceneTest.py, and it's not available in the multitude of other tests that write temp files. I'm all for making a unique path and ensuring it gets cleaned up, which is  why Gaffer provides `TestCase.temporaryDirectory()`. But until Cortex has an equivalent that can be used universally, I don't see the value in a half hearted effort. Perhaps I'm missing something about these files in particular? Does the ownership issue you mentioned in your other comment only apply to USD maybe? I'm happy to fix this up if its a specific problem, but otherwise I would suggest we instead prioritise a proper holistic cleanup at some point. The other nice things we could adopt from the Gaffer approach at the same time would be tests-as-modules and the ability to run one specific test in isolation.

Reply:
> I'm all for making a unique path and ensuring it gets cleaned up, which is why Gaffer provides `TestCase.temporaryDirectory()`. But until Cortex has an equivalent that can be used universally, I don't see the value in a half hearted effort.

I guess I was thinking that's how we got into this situation in the first place. It's difficult to prioritize the full effort, so trickling in improvements is the best we can hope for. For example, I'm not entirely sure why `TestCase.temporaryDirectory()` isn't already in Cortex, I don't think anyone would opposed that addition.

> Perhaps I'm missing something about these files in particular?

I guess I've just gotten in the habit of making this comment on all PRs lately. We had the problem in the entire IE codebase and the non-public projects have been largely cleansed lately.

> I would suggest we instead prioritise a proper holistic cleanup at some point. The other nice things we could adopt from the Gaffer approach at the same time would be tests-as-modules and the ability to run one specific test in isolation.

Those would be great improvements. Is that something your team is able to prioritize?